### DISCUSSION TITLE
Anyone have Visual Studio 2022 or later

### DISCUSSION THREAD

**Discussion by rabbibotton** [rabbibotton/clog#226] [General]
I am working on native apps for CLOG using WebView.h as the base. The first part is a generic executable clogframe (see clogframe/README.md)

If anyone has Visual Studio 2022 and can help out with the windows side give me a reply :)

clogframe is a very simple app and should be straight forward with instructions there to get working.

**Comment by b-tach** [comment]
Relatively late reply, but if this is still a need I will be glad to help!

**Reply by rabbibotton** [reply]
yes it is :) I need to create a windows ready version of clog/clogframe/clogframe.cpp (and if reasonable a make-windows file perhaps) using instructions from https://github.com/webview/webview

If you have the know how that would be awesome :)

**Reply by b-tach** [reply]
Well I was going to write a witty reply describing how I _don't_ have the know-how but will bang on the keyboard like my fellow 10,000,000 monkeys are. However, I did accidentally manage to figure it out in the end. Would you like me to commit the resulting clogframe.{obj, exe} as well as the build script I used (as you mentioned creating a Windows-ready version)?

**Reply by rabbibotton** [reply]
sorry for late answer, yes!!
